src/lib/components/molecules/waffled-squares/index.jsx

```diff
@@ -0,0 +1,117 @@
+import { useState } from "react";
+import { SvgSquare } from '$particles/svg-square'
+
+export const WaffledSquare = ({
+  name = 'Germany',
+  squaresTotal = 96,
+  waffleWidth = 100,
+  waffleHeight = waffleWidth,
+  rowLong = 10,
```

Comment:
Naming is famously one of the hardest things in programming. I think the name of this variable could be more descriptive, e.g. `rowLength` or `numberOfSquaresInRow`.